packages/e2e-tests/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM node:20.11-bullseye-slim
+
+ARG uid
+ARG gid
+
+# git is needed for gatherBuildInfo.js in build process
+RUN apt update && apt upgrade -y && apt install -y git
+
+# add a user "noone" with the same uid and gid as the host user
+RUN groupadd -g $gid -o noone
+RUN useradd -ms /bin/bash -u $uid -g $gid noone
```

Comment:
I can see that this now doesn't override `node_modules`. Very nice. But still, what files does this write? Why do we need this? Is this the HTML report file that Playwright generates? Wouldn't it be better to keep it in the container instead of writing to host? Using the `[outputFolder` option](https://playwright.dev/docs/test-reporters#html-reporter).